src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
```

Comment:
**suggestion**: Add type annotations for event handlers to enhance type safety and readability.